sysboundary/conductingsphere.cpp

```diff
@@ -184,29 +176,23 @@ namespace SBC {
                if(getR(cellCenterCoords[0],cellCenterCoords[1],cellCenterCoords[2],this->geometry,this->center) < this->radius) {
                   technicalGrid.get(i,j,k)->sysBoundaryFlag = this->getIndex();
                }
-
             }
          }
       }
-
-      return true;
    }
 
-   bool Conductingsphere::applyInitialState(
-      const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
-      FsGrid< std::array<Real, fsgrids::bfield::N_BFIELD>, FS_STENCIL_WIDTH> & perBGrid,
-      Project &project
-   ) {
-      const vector<CellID>& cells = getLocalCells();
-      #pragma omp parallel for
-      for (uint i=0; i<cells.size(); ++i) {
-         SpatialCell* cell = mpiGrid[cells[i]];
-         if (cell->sysBoundaryFlag != this->getIndex()) continue;
-         
-         for (uint popID=0; popID<getObjectWrapper().particleSpecies.size(); ++popID)
-            setCellFromTemplate(cell,popID);
+   void Conductingsphere::applyInitialState(const dccrg::Dccrg<SpatialCell, dccrg::Cartesian_Geometry> &mpiGrid,
+                                      FsGrid<std::array<Real, fsgrids::bfield::N_BFIELD>, 2> &perBGrid, Project &project) {
+      vector<CellID> cells = mpiGrid.get_cells();
+   #pragma omp parallel for
```

Comment:
So what is the practice for pragmas? I've seen no indentation, same as the next line, but never seen "one level back"...

Reply:
ah, that one slipped through. I wanted it on the same level. Fixed.